dvc/path_info.py

```diff
@@ -51,9 +50,7 @@ def fspath(self):
     url = fspath
 
     def relpath(self, other):
-        return self.__class__(
-            os.path.relpath(fspath_py35(self), fspath_py35(other))
-        )
+        return self.__class__(relpath(fspath_py35(self), fspath_py35(other)))
```

Comment:
Here are `fspath_py35()`, which should be removed once `relpath()` works with path-like objects.

Reply:
sorry for that, fixing